processor/src/system/mod.rs

```diff
@@ -2,6 +2,11 @@ use super::{
     ExecutionError, Felt, FieldElement, StarkField, SysTrace, Vec, Word, EMPTY_WORD, ONE, ZERO,
 };
 
+use core::{
```

Comment:
minor nit: I would remove a blank line above this import.

Reply:
done